admin/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="payment">
+            <group id="liqpay_section" sortOrder="6" showInDefault="0" showInWebsite="0" showInStore="0">
```

Comment:
please use pronko_liqpay id to avoid overlap with the other extensions.

Reply:
All these values should be set to "1"

showInDefault="1" showInWebsite="1" showInStore="1"